### ISSUE TITLE
[Zoom] RangeError: Invalid time value when opening upcoming meetings

### ISSUE THREAD

**Issue by IGassmann** [raycast/extensions#2721]
# Extension – [Zoom](https://github.com/raycast/extensions/tree/2e371837b4ea93b23bddec1d56729a689082be3b/extensions/zoom/)
Author: @thomaslombart

## Description

When running the "Upcoming Meetings" command I get the following error:

## Error:

P.S. I had to truncate line 66 because it was too long to fit in this GitHub issue.

```
RangeError: Invalid time value

Ez:upcoming-meetings.js:66:62949

---
63:       ${e>0&&e<1?`<path d="${SP(50,50,s,0,e*360)}" stroke="${t}" stroke-width="${a}" fill="none" />`:""}
64:     </svg>
65:   `.replaceAll(`
66: `,"")}`}cu.getProgressIcon=kP});var Gw=g(Lr=>{"use strict";var MP=Lr&&Lr.__createBinding||(Object.create?func...
67: Content-Disposition: form-data; name="`;return e.forEach((a,u)=>typeof a=="string"?A.push(n+eB(u)+`"\r
68: \r
69: ${a.replace(/\r(?!\n)|(?<!\r)\n/g,`\r
---

Ioe:upcoming-meetings.js:79:3634
upcoming-meetings.js:79:4063
    at Array.reduce (<anonymous>)
BH:upcoming-meetings.js:79:4024
upcoming-meetings.js:79:7545
Object.useMemo:react-reconciler.production.min.js:5:27195
exports.useMemo:react.production.min.js:25:208
Coe:upcoming-meetings.js:79:7540
Ql:react-reconciler.production.min.js:5:20846
```

### Raycast version
Version: 1.39.1

**Comment by mathieudutour** [comment]
do you perhaps have a meeting which somehow doesn't have start time?

**Comment by thomaslombart** [comment]
@IGassmann Do you happen to have recurring meetings in your upcoming meetings? I was able to reproduce the bug with this kind of meetings.

**Comment by IGassmann** [comment]
@thomaslombart Yes, I do. Most of them actually.

**Comment by thomaslombart** [comment]
That's the culprit. I assumed every meeting had a start time which is not the case with these types of recurring meetings. I'll fix that bug soon.

**Comment by thomaslombart** [comment]
The issue should be fixed with the following PR: #2757.
